### ISSUE TITLE
Generic.Formatting.MultipleStatementAlignment.NotSame doesn't parse short list properly after upgrading to 3.2.3

### ISSUE THREAD

**Issue by lcobucci** [squizlabs/PHP_CodeSniffer#1988]
After updating to `v3.2.3` this sniff is complaining about incorrect spacing regarding this code:

```php
<?php
declare(strict_types=1);

$payload = $message->getPayload();

[$userInput, $errors] = $this->normalizeUserInput($payload);
```

Output:

```sh
$ phpcs aa.php 
E 1 / 1 (100%)



FILE: aa.php
------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------
 4 | ERROR | [x] Equals sign not aligned with surrounding assignments; expected 14 spaces but found 1 space
------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------------------
```

And it doesn't complain anymore if we use `list()` instead of `[]`, like:

```php
<?php
declare(strict_types=1);

$payload = $message->getPayload();

($userInput, $errors) = $this->normalizeUserInput($payload);
```

Output:

```sh
$ phpcs aa.php 
. 1 / 1 (100%)


Time: 131ms; Memory: 14Mb
```

**Comment by lcobucci** [comment]
Maybe these are related: #1911 #1984

**Comment by jrfnl** [comment]
@lcobucci Hi Luís, #1870, #1848, #1924 are all related to this, but I'm surprised you are seeing the message as your example code looks to use blank lines between the lines and that would "break" a multi-statement block.
Could it be that the code you are testing on does not have the blank lines ?

So, this would trigger the notice:
```php
<?php
declare(strict_types=1);

$payload = $message->getPayload();
[$userInput, $errors] = $this->normalizeUserInput($payload);
```

This would not:
```php
<?php
declare(strict_types=1);

$payload = $message->getPayload();

[$userInput, $errors] = $this->normalizeUserInput($payload);
```

**Comment by jrfnl** [comment]
Just ran some tests which confirm what I said above.
Also: whether you use the short list syntax or the `list()` construct - correctly - has no influence on the results according to my test.

**Comment by lcobucci** [comment]
@jrfnl the code is exactly the way I've posted (with the line in between), which indeed surprised me. It can sure be an influence of another sniff but I'm afraid it wouldn't have that output while only analysing the code.

**Comment by jrfnl** [comment]
@lcobucci Hmm... the blank line in between should ensure that this sniff isn't triggered.
Hang on... ok.. so I've tested this now on `3.2.3` and can confirm your findings. Looks like this has already been fixed in `master` via #1924 which fixed #1922.

**Comment by lcobucci** [comment]
@jrfnl let me try it with `master`

**Comment by lcobucci** [comment]
@jrfnl it's indeed fixed in `master`, we can close it as a duplicate then. Thanks for your help 👍

**Comment by jrfnl** [comment]
You're welcome.
